src/index.js

```diff
@@ -0,0 +1,129 @@
+import './index.css';
+/* eslint-disable import/prefer-default-export */
+import { Display } from './modules/display.js';
+import {
+  addLikes, getAllLikes, getAllMovies, addComment, getAllCommentsByItemId,
+} from './modules/service.js';
+import Comment from './modules/comment';
+
+await Display();
+
+const shows = await getAllMovies();
+const modal = document.querySelector('.modal');
+const closeButton = document.querySelector('.close-button');
+const image = document.getElementById('img-detail');
+const btnModalComments = document.querySelectorAll('#modal-comments');
+const total = document.querySelectorAll('.likeId');
+const btnLikes = document.querySelectorAll('#btnLikes');
+const formComments = document.querySelectorAll('.formComment');
+const totalAddedComments = document.querySelector('#totalComments');
+const showComments = document.querySelector('#showComments');
+const itemDetails = document.querySelector('#itemDetails');
+let formId = 0;
+
+// START LIKES SECTION
+
+export const showLikes = async (showId) => {
+  let totalLikes = '';
+  let like = [];
+  const likes = await getAllLikes();
+
+  like = likes.filter((l) => l.item_id === showId);
+
+  if (like[0]) {
+    totalLikes = like[0].likes;
+  }
+  return totalLikes;
+};
+
+btnLikes.forEach((button) => {
+  button.addEventListener('click', (event) => {
+    event.preventDefault();
+    const id = parseInt(event.target.getAttribute('data-showid'), 10);
+    addLikes(id);
+
+    total.forEach((el) => {
+      showLikes(id).then((totalLikes) => {
+        el.innerHTML = `<br><br><span class="likes_count">Likes(${totalLikes})`;
+      });
+    });
+  });
+});
+
+// END LIKES SECTION
```

Comment:
- Hey, you've done a great job with this project however, the like seems to have some unexpected behavior. 
If I like a particular movie, the likes counter populates the likes for other movies even when I didn't like the other movies.
Also, the counter does not increment well as it seems to generate some random values. Please try fixing this.